### ISSUE TITLE
portals cannot be activated in vacuum - By MisterE

### ISSUE THREAD

**Issue by ExeVirus** [MeseCraft/MeseCraft#4]
portals can be built in air, but not in space

**Comment by AndyIII** [comment]
My solution was to build a dome with air in it, then light the portals. Works fine. This could be left this way to make this portion of the game harder. Would be annoying though if you let your portal turn off when you are on the space side.

**Comment by MisterE123** [comment]
to me, it seem expected behavior is that portals will activate in any medium

**Comment by MeseCraft** [comment]
No, it shouldn't be like this. I think portals just dont account for something other than air being present. 

It could be water, lava, vacuum, or other traversible matter that is on the other end of a portal, but a portal should still activate despite the hazards. I just got done watching all of StarGate SG1 and Atlantis so I feel like if anything, we should just adjust portals to account for more possible materials or maybe not specify.
